changelog.d/4989.misc

```diff
@@ -0,0 +1 @@
+Remove presence list support in accordance with spec.
```

Comment:
should probably be a feature rather than a misc. A ref to the MSC wouldn't hurt either.